availability-oracle/src/main.rs

```diff
@@ -132,6 +120,16 @@ struct Config {
         help = "a comma separated list of the supported data source kinds"
     )]
     supported_data_source_kinds: Vec<String>,
+
+    #[structopt(
+        long,
+        env = "REWARDS_MANAGER_CONTRACT",
+        help = "The address of the rewards manager contract"
+    )]
+    pub rewards_manager_contract: String,
```

Comment:
Might be convenient to parse this into an `Address` upfront.
```suggestion
    pub rewards_manager_contract: Address,
```

Reply:
Nice, updated!